showcase/app/styles/showcase-pages/dropdown.scss

```diff
@@ -23,6 +23,7 @@ body.components-dropdown {
 
   .shw-component-dropdown-fixed-height-container {
     min-height: 200px;
+    outline: 1px dotted #d3d3d3;
```

Comment:
do we have variables for these colors somewhere?

Reply:
not yet for the outline/borders, but we do have [a minimal set](https://github.com/hashicorp/design-system/blob/main/showcase/app/styles/_tokens.scss) of token in `showcase`. can take some time to abstract `#d3d3d3` too

Reply:
addressed in ff16764 by replacing with the closest grey available as token

Reply:
Cristiano asked me to move this in a separate PR so https://github.com/hashicorp/design-system/pull/2540 is the nbetter place to look for this change

Reply:
In the past we haven't worried so much about standardizing color values like this in the Showcase. (Just to mention, not saying we shouldn't.)

Reply:
indeed, `showcase` was generally seen as a scrappy playground